Dispatcher/UI/CustomButton.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  CustomButton.swift
+//  Dispatcher
+//
+//  Created by Hadar Basson on 28/08/2023.
+//
+
+import UIKit
+
+class CustomButton: UIButton {
```

Comment:
we need a better name to describe this button
there are alot of custom buttons

also this should be inside "Components" folder in UI